core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -132,19 +123,16 @@ public Long getFieldValue(Inode<?> o) {
   private final ContainerIdGenerable mContainerIdGenerator;
   private final InodeDirectoryIdGenerator mDirectoryIdGenerator;
 
-  /**
-   * This is only used for adding inodes from the journal, to prevent repeated lookups of the same
-   * inode.
-   */
-  private InodeDirectory mCachedInode;
-
   /**
    * @param containerIdGenerator the container id generator to use to get new container ids
    * @param directoryIdGenerator the directory id generator to use to get new directory ids
    * @param mountTable the mount table to manage the file system mount points
    */
   public InodeTree(ContainerIdGenerable containerIdGenerator,
       InodeDirectoryIdGenerator directoryIdGenerator, MountTable mountTable) {
+    mTtlBuckets = new TtlBucketList();
```

Comment:
why does `InodeTree` own this, and not `InodeTreePersistentState`?

Reply:
The TTL bucket list is modified by the InodeTtlChecker, and we don't use journal entries to track what is in the list.

I think we could change it so that InodeTtlChecker doesn't directly modify the TtlBucketList, and then have `InodeTreePersistentState` be the owner. This is non-trivial, so I'll do it in either a later PR in this feature branch, or in master after merging the branch. Do you have a preference for where to make the change? I'd lean towards master since there will be less code flying around that way.

Reply:
I see. We can change that after the merging this branch. Let's leave a TODO somewhere.

Reply:
added a TODO on the field in InodeTreePersistentState